server/server_test.go

```diff
@@ -1,94 +1,148 @@
 package server
 
 import (
+	"encoding/json"
 	"fmt"
 	"net/http"
 	"net/http/httptest"
 	"net/url"
+	"os"
 	"testing"
 
 	"io/ioutil"
 
 	"github.com/gin-gonic/gin"
+	"github.com/intervention-engine/fhir/models"
+	"github.com/intervention-engine/fhir/server"
+	"github.com/mitre/ptmerge/testutil"
 	"github.com/stretchr/testify/suite"
 )
 
 type ServerTestSuite struct {
-	suite.Suite
-	Engine *gin.Engine
-	Server *httptest.Server
+	// The MongoSuite from IE provides useful features for starting/stopping
+	// a test mongo database, as well as inserting test fixtures.
+	testutil.MongoSuite
+	PTMergeServer *httptest.Server
+	FHIRServer    *httptest.Server
```

Comment:
You've integrated the FHIRServer into the tests, but I don't think we can really know if it works because the PtMatch code never actually queries the FHIR server, right?  I assume you're just setting this up in anticipation of future capabilities?